content_scripts/fill-username-and-password.js

```diff
@@ -66,7 +66,13 @@ function writeValueToInputElement(element, value) {
   }
 }
 
-browser.runtime.onMessage.addListener(function _func (request, sender, sendResponse) {
+// Polyfill does not have this feature, so we need to check if we are on chrome
+var browserVar = browser;
```

Comment:
Maybe something like `browserVar = chrome || browser` could be clearer here?
Also, `let` may be more appropriate